src/common/config.cpp

```diff
@@ -174,6 +177,40 @@ bool wxConfigBase::Read(const wxString& key, int *pi, int defVal) const
     return r;
 }
 
+// size_t is stored either as long or long long (Win64)
+#if SIZEOF_SIZE_T == SIZEOF_LONG
```

Comment:
Sorry for nitpicking, but as `size_t` is unsigned, should we not used `unsigned long` and `wxULongLong` or `wxUint64` here?

But perhaps the compilers will not complain...

Reply:
Handling of unsigned types in `wxConfig` is suspicious generally speaking, but I didn't want to change it now, so I've just done the strict minimum needed to allow using `size_t` with it at all.